### PR TITLE
Generate SBOM on Windows with OS info and KBs

### PR DESCRIPTION
**Pull request by lebauce**
### What does this PR do?

Collects KB and system info on Windows and send it in a SBOM

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided. Except if the `qa/skip-qa` label, with required either `qa/done` or `qa/no-code-change` labels, are applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by sgnn7 — APPROVED** [review]
LGTM for ASC-owned `tasks/agent.py`

**Review by safchain — APPROVED** [review]

File: pkg/collector/corechecks/sbom/check.go [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2022-present Datadog, Inc.
 
-//go:build trivy
+//go:build trivy || windows
```
Comment (AliDatadog) [inline pkg/collector/corechecks/sbom/check.go:6]: Could we add the trivy build flag to windows instead ? I think the `windows` build flag can be used by the cluster-agent on windows

**Review by AliDatadog — APPROVED** [review]
Approved, as discussed it will need a new `wmi` flag

**Review by git-thuerk-done — APPROVED** [review]
good for docs

**Review by pducolin — APPROVED** [review]

**Review by chouetz — APPROVED** [review]

**Pull request merged**